packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -239,7 +239,7 @@ const ResetPassword = ({
             type="submit"
             className="cta-primary cta-xl"
           >
-            Begin Reset
+            Begin reset
```

Comment:
👍

Reply:
I was curious why this change but no .ftl change, but looks like the hardcoded version was out of sync w/ FTL file:

```sh
git grep -n "reset-password-button" packages/ | cat

packages/fxa-settings/src/pages/ResetPassword/en.ftl:14:reset-password-button = Begin reset
packages/fxa-settings/src/pages/ResetPassword/index.tsx:236:        <FtlMsg id="reset-password-button">
packages/fxa-settings/src/pages/ResetPassword/index.tsx:238:            data-testid="reset-password-button"
```